app/services/search/twitter_service.rb

```diff
@@ -1,38 +1,47 @@
 module Search
   class TwitterService
-    def self.find_tweets_for(event, client = $twitter_client)
-      last = event.items.last.try(&:id)
+    class << self
+      def find_tweets_for(event, client = $twitter_client)
+        last = event.items.last.try(&:id)
 
-      client.search("#{event.hash_tag} -rt", result_type: :recent).each_with_index do |tweet, i|
-        break if tweet.id.to_s.eql?(last) || i.eql?(500)
+        client.search("#{event.tag} -rt", result_type: :recent).each_with_index do |tweet, i|
+          break if tweet.id.to_s.eql?(last) || i.eql?(500)
 
-        event.items << make_a_item_by(tweet)
+          begin
+            event.items << make_a_item_by(tweet)
+          rescue Exception => e
```

Comment:
Avoid rescuing the `Exception` class. Perhaps you meant to rescue `StandardError`?